airflow/providers/docker/operators/docker.py

```diff
@@ -167,7 +167,8 @@ def __init__(
         shm_size: Optional[int] = None,
         tty: Optional[bool] = False,
         cap_add: Optional[Iterable[str]] = None,
-        **kwargs,
+        extra_hosts: Optional[Dict[str, str]] = None,
+            **kwargs,
```

Comment:
Invalid indentation?

Reply:
Yes, fixed and also rerun black on test code